X16 Reference - 11 -  Working with CMDR-DOS.md

```diff
@@ -0,0 +1,368 @@
+# Chapter 11: Working With CMDR-DOS
+This manual describes Commodore DOS on FAT32, aka CMDR-DOS. 
+
+## CMDR-DOS
+
+Commander-16 duplicates and extends the programming interface used by Commodore's line of disk drives, including the 
+famous (or infamous) VIC-1541. CMDR-DOS uses the industry-standard FAT-32 format. Partitions can be 32MB up to 
+(in theory) 2TB and supports CMD-style partitions, subdirectories, timestamps and filenames up to 255 characters. 
+It is the DOS built into the [Commander X16](https://www.commanderx16.com).
+
+There are three basic interfaces for CMDR-DOS: the binary interface (SAVE,LOAD), the data file interface (OPEN, 
+PRINT#, INPUT#, GET#), and the command interface. We will give a brief summary of BASIC commands here, but please refer
+to the BASIC chapter for full syntax of each command. 
+
+If you are familiar with the SD2IEC or the CMD hard drive, navigating partitions and subdirectories is similar, with "CD", "MD", and "RD" commands
+to navigate directories.
+
+## Binary Load/Save
+
+The primary use of the binary interface is loading and saving program files and loading binary files into RAM. 
+
+Your binary commands are LOAD, SAVE, BLOAD, VLOAD, and BVLOAD. 
```

Comment:
VERIFY and BVERIFY should now be working on both sdcard and host fs

Reply:
Added to the list here, but didn't actually add docs here. Users can refer to the BASIC docs for full details.